packages/react-router-dev/typegen/index.ts

```diff
@@ -71,11 +72,21 @@ async function createContext({
   };
 }
 
+function isRouteInAppDirectory(ctx: Context, route: RouteManifestEntry) {
+  const absoluteRoutePath = Path.resolve(ctx.config.appDirectory, route.file);
```

Comment:
`route.file` [is relative to the appDirectory](https://github.com/remix-run/react-router/blob/226dbbdf8f3d6fb1483a995cc321a1ca2a6b882c/packages/react-router-dev/config/routes.ts#L52-L56).